### PR TITLE
Integrate redux-persist

### PR DESCRIPTION
**Pull request by IViol**
JWWEB-89 redux-persist integration

### CODE REVIEW

**Comment by Romanior** [comment]
👍 Resolve conflicts please

File: .stylelintrc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "extends": [
     "stylelint-config-jibrelnetwork"
-  ]
+  ],
+  "ignoreFiles": "**/*.js",
```
Comment (Romanior) [inline .stylelintrc]: unrelated change

File: flow-typed/style.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // @flow
 
 declare module CSSModule {
-    declare var exports: { [key: string]: string };
+  declare var exports: { [key: string]: string }
```
Comment (Romanior) [inline flow-typed/style.js]: unrelated change

File: package.json [hunk 3]
```diff
@@ -64,6 +66,7 @@
     "babel-plugin-transform-replace-object-assign": "0.2.1",
     "babel-preset-airbnb": "2.4.0",
     "babel-preset-es2015": "6.24.1",
+    "babel-preset-es2015-loose": "8.0.0",
```
[the remark this answers is not in the document]
Comment (IViol) [reply package.json]: https://github.com/localForage/localForage/blob/master/.babelrc#L2

File: package.json [hunk 4]
```diff
@@ -30,6 +30,7 @@
     "jibrel-contracts-jsapi": "0.4.0",
     "json-format": "1.0.1",
     "jwallet-web-keystore": "jibrelnetwork/jwallet-web-keystore#2ef16aa",
+    "localforage": "jibrelnetwork/localForage",
```
[the remark this answers is not in the document]
Comment (IViol) [reply package.json]: we planned to use our own indexedDB provider for redux-persist. I wanted to use localforage as base. Planned time was not enough. So I switched on the others tasks (as they have higher priority).

[the remark this answers is not in the document]
Comment (IViol) [reply package.json]: changed to origin localforage currently

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Romanior) [reply package.json]: we should not transpile files from `node_modules` as  they, in general, were already transpiled and designed for direct usage without any preparation.
This matter discussed in numerous topics on the net, e.g.
https://stackoverflow.com/a/31822668

For our modules we should move to company npm and include only them. For now I would suggest to exclude this package.
